contrib/README.md

```diff
@@ -58,6 +58,18 @@ Supported platforms: Linux (required tools: `sysfs`)
 
 **vicious.contrib.batproc**
 
+**vicious.contrib.btc**
+
+Provides current Bitcoin price in any currency by code (
+https://en.wikipedia.org/wiki/ISO_4217).
+Supported platforms: Linux, FreeBSD.
```

Comment:
it should mention `curl` and your lua library as dependencies.

Reply:
ok, i updated required and squash commits